doc/Settings.md

```diff
@@ -137,13 +137,15 @@ Microsoft Store App support in WinGet is currently implemented as an experimenta
    },
 ```
 
-### packagedAPI
+### directMSI
```

Comment:
>directMSI [](http://example.com/codeflow?start=4&length=9)

Did two branches get merged here? Don't see support for directMSI yet anywhere else.